assets/static/__mocks__/localStorageMock.js

```diff
@@ -0,0 +1,22 @@
+// source: https://github.com/facebook/jest/issues/2098
+
+var localStorageMock = (function() {
+    var store = {};
+
+    return {
+        getItem: function(key) {
+            return store[key] || null;
+        },
+        setItem: function(key, value) {
+            store[key] = value.toString();
+        },
+        clear: function() {
+            store = {};
+        }
+    };
+
+})();
+
+Object.defineProperty(window, "localStorage", {
```

Comment:
It doesn't look like this mock is working correctly, it seems to be undefined in tests.